CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Added
+
+- Pruning chart-operator helm release from app-operator side.
```

Comment:
```suggestion
- Delete chart-operator helm release and chart CR so it can be re-installed.
```

The changelog should explain why we're doing this. So I think it needs to say its so chart-operator can be re-installed.